frontend/packages/data-portal/app/components/Run/ViewerPage.tsx

```diff
@@ -308,13 +402,15 @@ const toggleAllDepositions = () => {
   updateState((state) => {
     const layers = state.neuroglancer?.layers || []
     const layersOfInterest = layers.filter((l) => l.type !== 'image')
-    const archived = layersOfInterest.some(
-      (l) =>
+    const archived = layersOfInterest.some((li) => {
+      const l = li as LayerWithSource
+      return (
         boolValue(l.archived, /* defaultValue = */ false) &&
-        boolValue(l.visible, /* defaultValue = */ true),
-    )
+        boolValue(l.visible, /* defaultValue = */ true)
+      )
+    })
     for (const layer of layersOfInterest) {
-      layer.archived = !archived
+      ;(layer as LayerWithSource).archived = !archived
```

Comment:
Is the ";" in the beginning of the line a typo?

Reply:
Funny enough it's from the formatter, see https://prettier.io/docs/rationale.html#semicolons

Reply:
As a nice side-effect of moving the types to the neuroglancer package these are gone now anyway